### PR TITLE
Patch 1.8.1

### PR DESCRIPTION
**Pull request by XFox111**
## Changelog
- Added Spanish translation (#37)

## PR Checklist
- [x] Change extension version in the manifest

### CODE REVIEW

File: _locales/es/messages.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+		"description": "Alternative text for options button in the pane"
+	},
+	"closePanel":
+	{
+		"message": "Cerrar",
+		"description": "Alternative text for close panel button"
+	},
+	"loadOnRestore":
+	{
+		"message": "Cargas pestañas al restaurar",
```
Comment (XFox111) [inline _locales/es/messages.json:29]: Possible typo. GT suggests me to change it to "Carga**r** pestañas al restaurar"

File: _locales/es/messages.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		"description": "Save collection action name. Used in the pane, extension context menu and manifest shortcuts"
+	},
+	"nothingSaved":
+	{
+		"message": "No tienes pestañas guardadas",
+		"description": "Placeholder for empty pane"
+	},
+	"removeTab":
+	{
+		"message": "Eliminar pestañas de colección",
```
Comment (XFox111) [inline _locales/es/messages.json:79]: Spanish-English translation of the string seems a bit off. If says "Remove collection tabs" and according to the GT it missing "la" article. So I guess it should be "Eliminar pestañas de la colección"

**Review by XFox111 — COMMENTED** [review]
@Wafflects Hi. Thanks for your contribution to the project!
Unfortunately, I don't know Spanish, so I only can rely on Google Translator. I've added some comments in the PR in places I consider questionable. Could you please check them and explain to me why they are correct (or update them if they are not)?

File: _locales/es/messages.json (hunk 1 above)
Comment (Wafflects) [reply _locales/es/messages.json:29]: That's a typeo, my bad

File: _locales/es/messages.json (hunk 2 above)
Comment (Wafflects) [reply _locales/es/messages.json:79]: Both are grammatically correct, is down to preference. With the "la" article it sounds more formal than without it.

**Comment by Wafflects** [comment]
After some digging I found how the feature has called in Edge, I changed some words to make it closer to the original version.
And fixed some minor issues.

[es.zip](https://github.com/XFox111/TabsAsideExtension/files/5169796/es.zip)
Images from the original feature https://www.solvetic.com/tutoriales/article/4358-guardar-grupos-ventanas-microsoft-edge-usar-mas-tarde/

**Comment by XFox111** [comment]
Awesome! The last thing is the description. [Here's the English version of the description](https://gist.github.com/XFox111/069a4d4eaf94b368bccfa4a56bef057f)

**Pull request merged**